packages/firestore/src/platform_browser/browser_platform.ts

```diff
@@ -72,4 +73,27 @@ export class BrowserPlatform implements Platform {
   btoa(raw: string): string {
     return btoa(raw);
   }
+
+  randomBytes(nBytes: number): Uint8Array {
+    if (nBytes <= 0) {
+      return new Uint8Array();
+    }
+
+    // Polyfill for IE and WebWorker
+    // eslint-disable-next-line no-restricted-globals
+    const crypto =
+      (this.window &&
+        (this.window.crypto || (this.window as any)['msCrypto'])) ||
+      (self && self.crypto);
+    const v = new Uint8Array(nBytes);
+    if (!!crypto) {
```

Comment:
Is this double exclamation mark intentional? If not, you may want to add a unit test that would fail if Math.random() is used unintentionally.

Reply:
This can probably just be `if(crypto)` since an if statement just needs a "truthy" value and not a boolean.

Reply:
Done.